akka-javasdk/src/main/scala/akka/javasdk/impl/eventsourcedentity/EventSourcedEntityImpl.scala

```diff
@@ -167,10 +167,6 @@ private[impl] final class EventSourcedEntityImpl[S, E, ES <: EventSourcedEntity[
             case Left(err) =>
               Future.successful(new SpiEventSourcedEntity.ErrorEffect(err))
             case Right((reply, metadata)) =>
-              val delete =
-                if (deleteEntity) Some(configuration.cleanupDeletedEventSourcedEntityAfter)
```

Comment:
good. I was also expecting a change of SpiSettings constructor. Doesn't that show up as deprecated?

```
  @deprecated("Define cleanupDeletedEntityAfter constructor parameter", since = "1.4.3")
  def this(eventSourcedEntitySnapshotEvery: Int, devMode: Option[SpiDevModeSettings])
```

Reply:
No it did not, these changes broke compilation, but that did not show up, I'll fix that as well.

Reply:
it could be because we have this:
```
@nowarn("msg=deprecated") //TODO remove deprecation once we remove the old constructor
```

Reply:
Done in ddd9cb95e3deabd72eab46ebffecc2739739b1a1 but it's a little bit weird with the ES + KV setting under ES in config?